packages/components/src/border-control/stories/index.story.tsx

```diff
@@ -16,7 +16,7 @@ import { BorderControl } from '..';
 import type { Border } from '../types';
 
 const meta: Meta< typeof BorderControl > = {
-	title: 'Components (Experimental)/BorderControl',
+	title: 'Components/BorderControl',
```

Comment:
Similar to #65586 should we consider fixing the default bool prop value in Storybook:


https://github.com/user-attachments/assets/b2af5552-c9cd-4853-9310-dfb75f39fbfb

by adding the `true` to the story for all boolean props that default to `true` in the hook:

```diff
diff --git a/packages/components/src/border-control/stories/index.story.tsx b/packages/components/src/border-control/stories/index.story.tsx
index 19938db5d0e..1bcb22c50d6 100644
--- a/packages/components/src/border-control/stories/index.story.tsx
+++ b/packages/components/src/border-control/stories/index.story.tsx
@@ -93,6 +93,9 @@ export const Default = Template.bind( {} );
 Default.args = {
        colors,
        label: 'Border',
+       enableAlpha: true,
+       enableStyle: true,
+       shouldSanitizeBorder: true,
 };

 /**
```

Reply:
Yep, makes sense. Done.

Reply:
For the protocol, this is another thing that should ideally happen automatically. Right now, it feels like a Storybook glitch.

Reply:
I remember looking into this years ago. There was something about Storybook picking up the default automatically in controls. Don't really remember the details though.

Reply:
Definitely a storybook bug